lib/cosmos/tools/handbook_creator/handbook_creator.rb

```diff
@@ -47,42 +47,25 @@ def initialize_actions
       @hide_ignored_action.statusTip = 'Do not include ignored items in command and telemetry handbooks'
       @hide_ignored_action.setCheckable(true)
       @hide_ignored_action.setChecked(false)
+
+      @copy_assets_action = Qt::Action.new('&Copy Assets to Output', self)
```

Comment:
I could see someone wondering why their output/handbook/assets directory isn't getting updated or how do I refresh the original contents. That's why I created this menu option.